src/Frontend.hs

```diff
@@ -204,7 +204,7 @@ aulaSpace space =
        Page.viewIdeas  space
   :<|> Page.viewIdea
   :<|> Page.editIdea
-  :<|> Page.commentIdea locSpace
+  :<|> Page.commentIdea
```

Comment:
I don't really like the fact we don't use/check parts of the url (here the space).